cli/src/pcluster/models/imagebuilder.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This module contains all the classes representing the Resources objects.
+# These objects are obtained from the configuration file through a conversion based on the Schema classes.
+#
+
+from typing import List
+
+from pcluster import utils
+from pcluster.config.extended_builtin_class import MarkedBool, MarkedTag
+from pcluster.models.cluster import Resource, Tag
+from pcluster.validators.ec2 import BaseAMIValidator
+
+# ---------------------- Image ---------------------- #
+
+
+class Volume(Resource):
+    """Represent the volume configuration for the ImageBuilder."""
+
+    def __init__(self, size: int = None, encrypted: bool = None, kms_key_id: str = None):
+        super().__init__()
+        self.size = size
+        self.encrypted = encrypted
+        self.kms_key_id = kms_key_id
+        self._default_value()
+        # TODO: add validator
+
+    def _default_value(self):
```

Comment:
The idea to group the set of the defaults together it's a nice one.
I'd rename the function to `_set_defaults()`

Reply:
Done